lib/components/Checkbox/Checkbox.docs.tsx

```diff
@@ -147,21 +147,21 @@ const docs: ComponentDocs = {
           to the <Strong>disabled</Strong> prop.
         </Text>
       ),
-      Example: ({ id, toggleState }) =>
+      Example: ({ id }) =>
         source(
           <Stack space="medium">
             <Checkbox
-              id={id}
+              id={`${id}_unchecked`}
               disabled={true}
               checked={false}
-              onChange={() => toggleState('checked')}
+              onChange={() => {}}
               label="Unchecked"
             />
             <Checkbox
-              id={id}
+              id={`${id}_checked`}
               disabled={true}
               checked={true}
-              onChange={() => toggleState('checked')}
+              onChange={() => {}}
```

Comment:
Might be worth keeping the `toggleState` here. If someone is copying and pasting this example, they'll likely conditionally, rather than permanently, disable the checkbox, so IMO the state should still be able to change.